armi/reactor/converters/uniformMesh.py

```diff
@@ -806,6 +1106,28 @@ def _calculateReactionRates(lib, keff, assem):
                 continue
             globalFluxInterface.calcReactionRates(b, keff, lib)
 
+    def updateReactionRates(self):
```

Comment:
this is not called or tested anywhere. i think i might understand what its used for, but can you add a little more clarification? something like:

useful for testing xyz to compare abc on a consistent mesh...